sonar-plugin-api/src/main/java/org/sonar/api/batch/bootstrap/ProjectDefinition.java

```diff
@@ -44,7 +44,7 @@
 
   public static final String TESTS_PROPERTY = "sonar.tests";
 
-  public static final String BUILD_DIR_PROPERTY = "sonar.buildDir";
```

Comment:
ProjectDefinition is part of the API. Don't remove stuff without deprecation notice.